src/main/java/karuna/karuna_backend/Controllers/VisitorMessageController.java

```diff
@@ -19,6 +23,11 @@ class VisitorMessageController {
 
     private final VisitorMessageService visitorMessageService;
 
+    @Operation(summary = "Create visitor message", description = "Getting body from json (body of request) and saving it to database as visitor message with current date")
+    @ApiResponse(
+            responseCode = "201",
+            description = "Visitor message content",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = VisitorMessageDto.class)))
```

Comment:
When you add swagger docs, add them for errors aswell, so if you had I believe BAD_REQUEST in exception handler, this  code should be implemented in Controller swagger documentation.

Reply:
Its automaticly added, idk why, but swagger knows about bad request in validation

Reply:
as we talk on slack, i improve code of your suggestions, u have right, automaticly generate description is not plenti